nexus/reconfigurator/simulation/src/system.rs

```diff
@@ -869,7 +883,18 @@ impl SimSystemBuilderInner {
 
         for blueprint in state.blueprints {
             let blueprint_id = blueprint.id;
-            match self.add_blueprint_inner(Arc::new(blueprint)) {
+
+            // TODO Should we store planning reports in
+            // `UnstableReconfiguratorState`? It would be Even More Unstable -
+            // the reports are stored as JSON blobs in the database, and there's
+            // no guarantee that all the reports in a given
+            // `UnstableReconfiguratorState` would be consistent with each other
+            // - they're really only intended for human / omdb consumption and
+            // debugging. For now, we just construct empty reports for all
+            // blueprints in the loaded state.
```

Comment:
I'm not sure what to do about this. I think it's fine to omit reports from `UnstableReconfiguratorState`; it also seems fine to include the JSON blobs (but not attempt to parse them anywhere, for reasons described by this comment) so we at least have them?

Reply:
I do think that's a little different.  I could see somebody stumbling on these and saying "oh, I'll write a `reconfigurator-cli` that parses this and uses that information to ... "